### PR TITLE
Add day 2 files

### PR DESCRIPTION
**Pull request by jlpalomino**
@lwasser These changes add the markdown files for the day 2 lessons on variables and lists in Python.

### CODE REVIEW

**Comment by lwasser** [comment]
thank you @jlpalomino !! at first glance travis is happy with the PR. i'll build locally and then merge :) today sometime. i have a few meetings starting at 10am but this will get done. thank you for your hard work on this course. it's coming together beautifully!

File: _posts/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-01-variables.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+* Write `Python` code to:
+    * create variables to store single data values (e.g. integers, decimals, text string) 
+    * run simple arithmetic calculations on these variables (e.g. with the addition operator `+`)
+   
+ 
+# <i class="fa fa-check-square-o fa-2" aria-hidden="true"></i> What You Need
+
+Be sure that you have followed the instructions on <a href="{{ site.url }}/workshops/setup-earth-analytics-python/setup-git-bash-anaconda/" target = "_blank">Setting up Git, Bash, and Anaconda on your computer</a> to install the tools for your operating system (Windows, Mac, Linux). 
+
+Be sure that you have completed the previous lesson on <a href="{{ site.url }}/courses/earth-analytics-bootcamp/get-started-with-open-science/jupyter-notebook-interface/" target = "_blank">The Jupyter Notebook Interface</a>.
```
Comment (lwasser) [inline _posts/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-01-variables.md]: @jlpalomino so here are link rules for our website.

1. if the link is EXTERNAL - this means that it takes you to another website, please open the link in a NEW BROWSER tab ie use: `target = _blank`
2. if the link is INTERNAL - thus it links to earthdatascience.org - anywhere on the site, please do not use `target=_blank` - the window can open on the same page. 

So for this PR - please update all links that are internal to earthdatascience.org to open in the same window - ie please remove target=_blank

File: _posts/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+![png]({{ site.url }}//images/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib_12_0.png)
+
+
+
+
+<div class="notice--info" markdown="1">
+
+## <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Additional Resources
+
+<a href="https://matplotlib.org/examples/pylab_examples/subplots_demo.html" target="_blank">More Matplotlib Examples</a>
```
Comment (lwasser) [inline _posts/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib.md]: This is good. please just list these as a list using 
`* `
so any additional resources box will look like this:

```
* <a href="https://matplotlib.org/examples/pylab_examples/subplots_demo.html" target="_blank">More Matplotlib Examples</a>
* <a href="https://realpython.com/blog/python/python-matplotlib-guide/?utm_campaign=Data%2BElixir&utm_medium=email&utm_source=Data_Elixir_172" target="_blank">In-depth Guide to Matplotlib</a>
* <a href="http://joseph-long.com/writing/colorbars/" target="_blank">More Information on Color Bars</a>
	
```

**Comment by lwasser** [comment]
@jlpalomino  i have requested 2 - (very small but 1 is important) changes.

1. IMPORTANT - please keep internal links on the same window! -- see my specific comment on one page - apply this throughout.
2. please add bullets to additional resource lists for readability and consistency!

Once you do that i'll merge this as it looks FANTASTIC!! Nice work.

**Comment by jenp0277** [comment]
@lwasser These edits have been committed and are available for review in this PR. Note that I will have to go back to the Day 1 lessons to update the internal links as well.

**Comment by lwasser** [comment]
@jenp0277  is this your gmail account? :) thank you!

**Comment by jlpalomino** [comment]
@lwasser Yes, sorry, I was still logged into Github.com as my test user. I'm back to me now.

**Comment by lwasser** [comment]
ok wonderful! merging this. Thank you @jlpalomino !!it should be live in about 10 minutes (or less)

**Comment by lwasser** [comment]
☺ got it,  thanks!

**Pull request merged**